src/platforms/common/enriching-events/context.mdx

```diff
@@ -21,7 +21,7 @@ If you need to be able to search on custom data, [use tags](../tags) instead.
 
 ## Structured Context
 
-The best practice to attach custom data is via structured contexts. A context must always be a dictionary or map, and its values can be arbitrary.
+The best practice to attach custom data is via structured contexts. A context must always be a dictionary, object or map (depending on the programming language you're using), and its values can be arbitrary.
```

Comment:
Should we just make this platform specific? E.g.:

```mdx
<PlatformSection supported={["javascript", "node"]}>
The best practice to attach custom data is via structured contexts. A context must always be an object, and its values can be arbitrary.
</PlatformSection>

<PlatformSection notSupported={["javascript", "node"]}>
The best practice to attach custom data is via structured contexts. A context must always be a dictionary or map, and its values can be arbitrary.
</PlatformSection>
```

No strong feelings, also OK this way but figured we may as well provide "correct" wording, and in this case the duplication is not too large. But feel free to discard!